route/identicon/index.js

```diff
@@ -0,0 +1,20 @@
+const express = require('express')
+const router = express.Router()
+
+const identiconCanvas = require('../../lib/identicon_canvas')
+
+router.get('/:address', function (req, res) {
+  const address = req.params.address;
+  const identiconDataUrl = identiconCanvas(address).toDataURL()
+
+  let el = `<div style="` +
+           `background-image: url('${identiconDataUrl}');` +
+           `background-size: 100%;` +
+           `width: 24px;` +
+           `height: 24px` +
+           `"></div>`;
```

Comment:
> Takes `GET` requests at `/identicon/<address>`. Returns string containing a 24 by 24 px div with the identicon as an inline background image.

@repoman Can we change the response type from `text/html` to `image/png` and then only return the `${identiconDataUrl}` in the response? (I.e. without any surrounding HTML code).

<img width="619" alt="Screenshot 2020-07-20 at 14 19 42" src="https://user-images.githubusercontent.com/15718174/87905883-1ab10200-ca94-11ea-88e7-c5ab39837f45.png">